cmd/nfd-master/main.go

```diff
@@ -107,6 +107,8 @@ func initFlags(flagset *flag.FlagSet) *master.Args {
 	flagset.BoolVar(&args.VerifyNodeName, "verify-node-name", false,
 		"Verify worker node name against the worker's TLS certificate. "+
 			"Only takes effect when TLS authentication has been enabled.")
+	flagset.StringVar(&args.NsFormat, "label-ns", "legacy",
+		"Feature label namespace format. By default set to legacy.")
```

Comment:
I think we need the possibility to have both short and long turned on at the same time 🤔 

One option is to make this flag tri-state, i.e. smth like `-label-ns={legacy,short,both}`. If keeping the flag as string parameter we need to do input validation of it.

Another way could be having separate bool flags, i.e. smth like `-label-ns-legacy` and `-label-ns-short`. In this case we probably want to check that at least one of them is turned on.